.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,47 +1,48 @@
 ---
 name: 🐛 Bug Report
-about: Create an issue report.
+about: Create a report to help us improve Strapi.
 ---
 
-<!--
-Hello 👋 Thank you for submitting an issue.
+> Hello 👋 Thank you for submitting an issue.
```

Comment:
We intentionally didn't want to show these, making them quotes means we will have a large block of text at the beginning of every issue instead of a hidden comment.

I don't think we should make this change

Reply:
I will make changes and return the previous state of these blocks

Reply:
@derrickmehaffy Done